src/components/GlobalLearningResourcesPage/GlobalLearningResourcesPage.tsx

```diff
@@ -1,22 +1,28 @@
-import React, { Suspense } from 'react';
+import React, { Suspense, useState } from 'react';
 import GlobalLearningResourcesHeader from './GlobalLearningResourcesHeader';
 import GlobalLearningResourcesTabs from './GlobalLearningResourcesTabs';
 import GlobalLearningResourcesFilters from './GlobalLearningResourcesFilters';
 import GlobalLearningResourcesContent from './GlobalLearningResourcesContent';
 import './GlobalLearningResourcesPage.scss';
 import useSuspenseLoader from '@redhat-cloud-services/frontend-components-utilities/useSuspenseLoader/useSuspenseLoader';
 import { Spinner } from '@patternfly/react-core';
-import fetchAllData from '../../utils/fetchAllData';
+import fetchAllData, { loaderOptionsDefault } from '../../utils/fetchAllData';
+import { FetchQuickstartsOptions } from '../../utils/fetchQuickstarts';
 
 export const GlobalLearningResourcesPage = () => {
   const { loader, purgeCache } = useSuspenseLoader(fetchAllData);
+  const [loaderOptions, setLoaderOptions] =
```

Comment:
@JetyAdam do we need this to be stored in root state for the entire `GlobalLearningResourcesPage` and passed down? Or could we perhaps use Jotai to store it as an atom and subscribe only where it is being used?

Reply:
I believe that we need this for the entire `GlobalLearningResourcesPage` to make sure that both filters and quickstarts are updated whenever one of them is modified. Even though Jotai is also an option, I do not think it is worth it in this case.